src/prefect/flow_engine.py

```diff
@@ -95,6 +97,7 @@ class FlowRunEngine(Generic[P, R]):
     flow_run_id: Optional[UUID] = None
     logger: logging.Logger = field(default_factory=lambda: get_logger("engine"))
     wait_for: Optional[Iterable[PrefectFuture]] = None
+    _result: Union[R, Exception, Type[NotSet]] = NotSet
```

Comment:
Everything in the PR looks awesome except I really want this to not have the word "result" in it haha.  `_flow_return_value`, I'm not sure....

Reply:
That word is a tad overloaded. I'll update to `_return_value` and add a comment for clarity.

Reply:
Woot!